.tekton/test-pipeline-service-deployment-newer-ocp.yaml

```diff
@@ -0,0 +1,40 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: test-pipeline-service-deployment-newer-ocp
```

Comment:
I agree. It will also avoid a "version dance" if/when we change supported version (i.e. 4.13 won't need to be replace in the original pipeline and 4.14 won't need to be added to the "newer" pipeline, the only thing that happens is a file is added/deleted).

Reply:
fixed.